### PR TITLE
Feature: Update `gasp-avs` operator setup manual according to latest updates GASP-1624

### PR DESCRIPTION
**Pull request by tenequm**
Revise the documentation to reflect the new configuration and operational details for the `gasp-avs` operator, including updated environment variables and service definitions.

GASP-1624

### CODE REVIEW

File: compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    volumes:
+      - ./data:/data
+      - ${ECDSA_KEY_FILE_HOST:-./operator_keys/ecdsa.key.json}:/app/operator_keys/ecdsa_key.json:readonly
+      - ${BLS_KEY_FILE_HOST:-./operator_keys/bls.key.json}:/app/operator_keys/bls_key.json:readonly
+    environment:
+      AVS_BLOCK_VALIDATION_PERIOD: ${AVS_BLOCK_VALIDATION_PERIOD:-100}
+      AVS_DEBOUNCE_RPC: ${AVS_DEBOUNCE_RPC:-20}
+      AVS_KICK_PERIOD: ${AVS_KICK_PERIOD:-25}
+      AVS_TASK_EXPIRATION: ${AVS_TASK_EXPIRATION:-300}
+      AVS_UPDATE_STAKE_PERIOD: ${AVS_UPDATE_STAKE_PERIOD:-150}
```
Comment (vanderian) [inline compose.yml]: these should be removed, these are for AGG not OP

File: compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+services:
+  gasp-avs:
+    image: gaspxyz/gasp-avs:7387cc389edd5792421456e72c2096b1712d83b5
+    restart: unless-stopped
+    env_file: .env
+    volumes:
+      - ./data:/data
+      - ${ECDSA_KEY_FILE_HOST:-./operator_keys/ecdsa.key.json}:/app/operator_keys/ecdsa_key.json:readonly
+      - ${BLS_KEY_FILE_HOST:-./operator_keys/bls.key.json}:/app/operator_keys/bls_key.json:readonly
```
Comment (vanderian) [inline compose.yml]: these are no longer in .env, we can remove the volumes
OP will get the keys from the JSONs

File: compose.yml (hunk 1 above)
Comment (tenequm) [reply compose.yml]: @vanderian, thanks for pointing that out!
Removed!

File: compose.yml (hunk 2 above)
Comment (tenequm) [reply compose.yml]: removed

**Review by vanderian — APPROVED** [review]

**Pull request merged**